src/hotspot/share/gc/g1/g1ConcurrentRefine.hpp

```diff
@@ -143,6 +143,8 @@ class G1ConcurrentRefine : public CHeapObj<mtGC> {
 
   uint64_t adjust_threads_period_ms() const;
   bool is_in_last_adjustment_period() const;
+  class RemSetSamplingClosure;
```

Comment:
```suggestion

  class RemSetSamplingClosure;
```

Maybe an added newline to emphasize that declaration.

Reply:
Done.